Dockerfile

```diff
@@ -6,7 +6,7 @@ COPY package*.json /usr/src/app/
 RUN npm install
 COPY . /usr/src/app
 
-CMD node example.js
+CMD node app.js
 
 EXPOSE 3000
 EXPOSE 5000/udp
```

Comment:
nit newline

Reply:
the extra line here is intentional as a way to show the main file.

Reply:
adding a newline at the end of the file will get rid of that red circle warning "No newline at end of file"

Reply:
Aha thanks